package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "TEAM-A",
+  "version": "1.0.0",
+  "description": "This is our Readme file. Let's build something!",
+  "main": "main.js",
+  "scripts": {
+    "start-dev": "npm run build-watch & npm run start-server",
+    "start-server": "nodemon main.js -e html,js,css --ignore public",
+    "build-watch": "webpack -w",
```

Comment:
missing a build script for your deploy. Heorku needs a build script to know how to read good and build your app. ie webpack